### DISCUSSION TITLE
Processing does not work

### DISCUSSION THREAD

**Discussion by loneblues** [MedicalImageAnalysisTutorials/SlicerCochlea#37] [Q&A]
Hello. I tried to use SlicerCochlea for Slicer 4.10.2 and 5.0.3 on my Windows 10 PC. 
I used a T2 3D TSE image of the inner ear, and I picked a cochlea location as in the documentation. 
However, when I hit run, ...please wai message does not go through, and no segmentation appears even after a long wait. 
Hope someone can give me some advice to make this segmentation work. 
Thanks.

**Comment by idhamari** [comment]
Hi, please provide the error message you see in the python console.
